decidim-elections/app/controllers/decidim/elections/admin/answers_controller.rb

```diff
@@ -57,6 +57,20 @@ def update
           end
         end
 
+        def select
+          MarkAnswerAsSelected.call(answer, current_user) do
```

Comment:
Should we add a permission or check that the voting phase has finished before selecting them?

Something like:

```
enforce_permission_to :select, :answer, election: election, question: question
```

Reply:
Yes, makes sense! I'll add it. 👍